src/components/ProjectPreview.jsx

```diff
@@ -5,14 +5,19 @@ import moment from 'moment';
 const MAX_LENGTH = 50;
 
 export default function ProjectPreview({preview, project}) {
+  const projectTimestamp = project.updatedAt ?
+    (
+      <div className="project-preview__timestamp">
+        {moment(project.updatedAt).fromNow()}
+      </div>
+    ) : null;
+
   return (
     <div>
       <div className="project-preview__label">
         {preview.slice(0, MAX_LENGTH)}
       </div>
-      <div className="project-preview__timestamp">
-        {moment(project.updatedAt).fromNow()}
-      </div>
+      {projectTimestamp}
```

Comment:
This is something of a moving target but at this point I’ve come around to trying to inline JSX conditionals whenever possible so that it reads more like a “template”, in this case something like:

```jsx
{project.updatedAt && (
  <div className="project-preview__timestamp">
    {moment(project.updatedAt).fromNow()}
  </div>
)}
```

Reply:
Yeah noticed this as well. Tried to follow how conditional JSX was written in other parts. Okay.